drivers/storage/port/scsiport/fdo.c

```diff
@@ -732,6 +777,16 @@ FdoDispatchPnp(
         {
             return FdoHandleDeviceRelations(portExt, Irp);
         }
+        case IRP_MN_QUERY_ID:
+        {
+           if (ioStack->Parameters.QueryId.IdType == BusQueryCompatibleIDs)
+           {
+               IoForwardIrpSynchronously(portExt->Common.LowerDevice, Irp);
```

Comment:
Set the `Irp->IoStatus.Information` to 0, in case `IoForwardIrpSynchronously` fails